pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -26,17 +23,6 @@ func NewResourceSyncController(
 		eventRecorder,
 	)
 
-	scheduler, err := configInformer.Config().V1().Schedulers().Lister().Get("cluster")
-	if err != nil {
-		klog.Infof("Error while listing scheduler %v", err)
-	}
-	if scheduler != nil && len(scheduler.Spec.Policy.Name) > 0 {
-		if err := resourceSyncController.SyncConfigMap(
-			resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: scheduler.Spec.Policy.Name},
-			resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalUserSpecifiedConfigNamespace, Name: "policy-configmap"}); err != nil {
```

Comment:
I agree these lines are wrong (namespace/name are switched) but do we want to remove them?

Target config controller does basically the same copy, but only when the scheduler CR is updated. Do we have a test for when the user creates a custom policy configmap (in `openshift-config`), adds it to the scheduler CR, then later updates that custom policy configmap directly?

I am wondering if, without this, changes will only be synced between `openshift-config` and the target namespace when the scheduler CR is updated (so really only by changing the custom policy configmap name)

Reply:
> Target config controller does basically the same copy, but only when the scheduler CR is updated. 

Are you sure about this? I tested it locally and the configmap got synced after I edited openshift-config one even without the lines.

Reply:
I actually wasn't sure, sorry, meant to pose that as more "did we check this?" based on talking about it offline the other day. If it works then cool! But do we have a test for that case?

Reply:
We do here https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/test/e2e/scheduler_test.go#L204